gpustack/cmd/start.py

```diff
@@ -666,6 +673,15 @@ def set_third_party_env(cfg: Config):
         os.environ["HF_HUB_DISABLE_XET"] = "1"
         logger.debug("set env HF_HUB_DISABLE_XET=1")
 
+    # Bridge config to env for existing env-based consumers
+    if cfg.external_auth_default_active is not None:
+        os.environ["GPUSTACK_EXTERNAL_AUTH_DEFAULT_ACTIVE"] = (
+            "true" if cfg.external_auth_default_active else "false"
+        )
+        logger.debug(
+            f"set env GPUSTACK_EXTERNAL_AUTH_DEFAULT_ACTIVE={'true' if cfg.external_auth_default_active else 'false'}"
+        )
+
```

Comment:
We can remove this part. Env is already handled in `default=get_gpustack_env_bool("EXTERNAL_AUTH_DEFAULT_ACTIVE"),`